modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/liferay-frontend.tld

```diff
@@ -601,6 +601,12 @@
 			<rtexprvalue>true</rtexprvalue>
 			<type>int</type>
 		</attribute>
+		<attribute>
+			<name>ctCollectionId</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+			<type>long</type>
+		</attribute>
```

Comment:
We don't need an attribute for `ctCollectionId` since you're setting the value directly on the request. If you want to do it through the tag attribute, you'll pass it in through the tag in https://github.com/liferay/liferay-portal/blob/master/modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/user_display_data.jsp#L75-L81.